### PR TITLE
Camel 3.19 - whats new.

### PR DESCRIPTION
**Pull request by davsclaus**
Feedback is welcome and to help update this blog with relevant content.

### CODE REVIEW

**Comment by oscerd** [comment]
I added the secret refresh part.

**Review by oscerd — APPROVED** [review]

File: content/blog/2022/10/camel319-whatsnew/index.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+
+### Rest DSL
+
+It is now possible to inline routes in Rest DSL which means that a Rest service corresponds to 1 route,
+and not 2 as before. This requires to enable `inline-routes=true` in Rest configuration,
+and to link the routes using `direct` endpoints.
+
+### Security
+
+We have upgraded to `TLSv1.3` as default, in components that are using SSL/TLS.
```
Comment (gzurowski) [inline content/blog/2022/10/camel319-whatsnew/index.md:53]:
```suggestion
We have upgraded to `TLSv1.3` as default in all components that are using SSL/TLS.
```

File: content/blog/2022/10/camel319-whatsnew/index.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+### New Components
+
+There are 5 new components:
+
+- camel-aws-cloudtrail - Consume events from Amazon Cloudtrail
+- camel-elasticsearch - Send requests to ElasticSearch via Java Client API
+- camel-hyperledger-aries - Camel support for Hyperledger Aries
+- camel-mapstruct - Type Conversion using Mapstruct
+- camel-python - To use python scripts
```
Comment (gzurowski) [inline content/blog/2022/10/camel319-whatsnew/index.md:82]:
```suggestion
- `camel-aws-cloudtrail` - Consume events from Amazon Cloudtrail
- `camel-elasticsearch` - Send requests to ElasticSearch via Java Client API
- `camel-hyperledger-aries` - Camel support for Hyperledger Aries
- `camel-mapstruct` - Type Conversion using Mapstruct
- `camel-python` - To use python scripts
```

**Review by gzurowski — APPROVED** [review]
LGTM 👍

File: content/blog/2022/10/camel319-whatsnew/index.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+    camel ps
+    PID   NAME                          READY  STATUS    AGE
+    61818  sample.camel.MyCamelApplica…   1/1   Running  26m38s
+    62506  dude                           1/1   Running   4m34s
+
+To see a bit more information, you can use `camel get`.
+
+Suppose you have two running processes, then you can manage them, such as stop,
+or stop/start routes etc:
```
Comment (essobedo) [inline content/blog/2022/10/camel319-whatsnew/index.md:29]:
```suggestion
Suppose you have two running integrations, then you can manage them, such as stop (with `camel stop <integration-name-or-pid>`),
or stop/start routes (with respectively `camel cmd stop-route --id=<id-route> <integration-name-or-pid>`/`camel cmd start-route --id=<id-route> <integration-name-or-pid>`) etc:
```

File: content/blog/2022/10/camel319-whatsnew/index.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+- `camel-hyperledger-aries` - Camel support for Hyperledger Aries
+- `camel-mapstruct` - Type Conversion using Mapstruct
+- `camel-python` - To use python scripts
+
+### Spring Boot
+
+We have upgraded to the latest Spring Boot 2.7 release.
+
+Camel Spring Boot now supports `platform-http` to make it easy to
+define Camel routes that starts from the embedded HTTP server inside Spring Boot.
```
Comment (essobedo) [inline content/blog/2022/10/camel319-whatsnew/index.md:89]:
```suggestion
define Camel routes that start from the embedded HTTP server inside Spring Boot.
```

File: content/blog/2022/10/camel319-whatsnew/index.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+**NOTE:**
+The context reloading feature is implemented as general functionality, which means, that its possible
+to use for other use-cases.
+
+### Camel Kafka
+
+We upgraded to Kafka Clients 3.2.x.
+Various other improvements and bug fixes as well, such as better error messages with health checks.
+
```
Comment (orpiske) [inline content/blog/2022/10/camel319-whatsnew/index.md:82]: ### Camel Telegram 

We migrated the external HTTP client used by this component to the HTTP client provided by the JVM.

**Comment by essobedo** [comment]
Not directly related but it makes me realize that we never made a blog post about what's new in 3.18, was-it intended?

**Review by essobedo — APPROVED** [review]

File: content/blog/2022/10/camel319-whatsnew/index.md (hunk 5 above)
Comment (davsclaus) [reply content/blog/2022/10/camel319-whatsnew/index.md:82]: Okay thanks

**Comment by davsclaus** [comment]
3.18 was released during summer and I was not around to draft a blog, and we didn't get started. Its a bit pity as its a LTS release.

**Pull request merged**